integrationtests/self/datagram_test.go

```diff
@@ -47,19 +46,23 @@ var _ = Describe("Datagram test", func() {
 
 			if expectDatagramSupport {
 				Expect(conn.ConnectionState().SupportsDatagrams).To(BeTrue())
-
 				if enableDatagram {
+					f := &wire.DatagramFrame{DataLenPresent: true}
+					maxDatagramMessageSize := f.MaxDataLen(num, conn.ConnectionState().Version)
```

Comment:
`num` is the concurrency parameter here. Can we use a different value? Maybe just set it to 250 bytes or something like that.

Reply:
Done.